lib/reporters/html/view-model.js

```diff
@@ -7,11 +7,12 @@ const TestCounter = require('../../test-counter');
 const NO_STATE = 'NO_STATE';
 
 module.exports = class ViewModel {
-    constructor(config) {
+    constructor(config, reporterOptions) {
         this._tree = {name: 'root'};
         this._skips = [];
         this._counter = new TestCounter();
         this._config = config;
+        this._reporterOptions = reporterOptions || {};
```

Comment:
You did it here - https://github.com/gemini-testing/gemini/pull/751/files#diff-6d7b77a01a322ffa7a900fd58c0a1ce4R130